src/main/kotlin/frc/robot/autonomous/AutoCommands.kt

```diff
@@ -91,38 +104,38 @@ fun alignToPose(
     )
 
     return Commands.run(
-            {
-                val robotPose = robotPoseSupplier.get()
-                val targetPose = targetPoseSupplier.get()
+        {
+            val robotPose = robotPoseSupplier.get()
+            val targetPose = targetPoseSupplier.get()
 
-                val radius =
-                    (robotPose.translation - targetPose.translation).norm
-                val output = radialController.calculate(radius, 0.0)
-                val fieldRelativeAngle =
-                    targetPose.translation.rotationToPoint(
-                        robotPose.translation
-                    )
-                val targetSpeeds =
-                    ChassisSpeeds(
-                        output * fieldRelativeAngle.cos,
-                        output * fieldRelativeAngle.sin,
-                        rotationController.calculate(
-                            robotPose.rotation.radians,
-                            targetPose.rotation.radians
-                        )
+            val radius =
+                (robotPose.translation - targetPose.translation).norm
+            val output = radialController.calculate(radius, 0.0)
```

Comment:
It's better to set the setpoint once (`radialController.setpoint = 0.0`) then use `calculate()` without the setpoint.